### ISSUE TITLE
BaseFileChooser and FileDrop don't escape special characters/whitespace

### ISSUE THREAD

**Issue by myegorov** [chriskiehl/Gooey#75]
When setting textbox value, BaseFileChooser (widget_pack.py) truncates filename on whitespace, and doesn't escape special characters (quotes, parentheses...). Same problem with the dragging & dropping in FileDrop (filedrop.py). This occurs on Linux.

**Comment by chriskiehl** [comment]
Examples please.

**Comment by myegorov** [comment]
Assuming chriskiehl/Gooey#76 issue corrected with `self.text_box.SetValue(result)`, here's what happens when I browse to (or drag & drop) a file with
```python
parser = gooey.GooeyParser()
parser.add_argument("filename", metavar="FILE", widget='FileChooser')
filename = parser.parse_args().filename
```
For a file name like `John's_artwork.txt`:
```
/bin/sh: 1: Syntax error: Unterminated quoted string
```

For a file name like `Johns(artwork).txt`:
```
/bin/sh: 1: Syntax error: "(" unexpected
```

A file name like `Johns artwork.txt`:

```
usage: tmpeLgPfo.py [-h] FILE
tmpeLgPfo.py: error: unrecognized arguments: artwork.dwg
```

Special characters in file names can be escaped for example with `string.translate` and `string.maketrans`.

Thanks!

**Comment by chriskiehl** [comment]
I'm unable to reproduce this on my Ubuntu or Windows machines. This sounds exactly like the issue @amorde fixed in commit 2b61019f4. Are you sure you're on the latest version? 

I'd say clone a fresh copy, give it shot, and let me know what happens.

**Comment by myegorov** [comment]
Yep, this is it.
